visualizations/frontend/other/TTable.vue

```diff
@@ -1352,7 +1352,7 @@ export default {
           urlparam: filterName,
         });
       }
-      this.setFilterValue(filterName, filterValue);
+      this.setFilterValue(filterName, filterValue, undefined, true);
```

Comment:
After reviewing all the code related to `setFilterValue` I think that we are at a level of need to go from single-ordered arguments to one single object argument signature.
I do not expect this to be done on these PRs but we should create a ticket so that we do not forget about it:

In `topcoat-core/visualization_mixin`, current `setFilterValue` signature:
```js
setFilterValue(name, value, defaultValue, skipOnReset=false)
```
proposal:
```js
setFilterValue({name, value, defaultValue, skipOnReset: false})
```

Reply:
https://snyksec.atlassian.net/browse/DP-776